plugins/pkg_commands.d/centos

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+. /etc/os-release
+
+# CentOS uses dnf or rpm-ostree(edge images) as the package command
+if [[ -x /usr/bin/rpm-ostree ]]; then
+    RSTRNT_PKG_CMD=${RSTRNT_PKG_CMD:-rpm-ostree}
+    RSTRNT_PKG_CMD=${RSTRNT_PKG_ARGS:-A --idempotent --allow-inactive}
+else
+    RSTRNT_PKG_CMD=${RSTRNT_PKG_CMD:-dnf}
```

Comment:
Actually, i think this can be dropped, because default is yum. which is supported on centos by default. Which im guessing is the reason this has never been added